packages/firebase_remote_config/ios/Classes/FirebaseRemoteConfigPlugin.m

```diff
@@ -0,0 +1,167 @@
+                        initWithLong:(long)[[remoteConfig lastFetchTime] timeIntervalSince1970] *
+                                     1000];
+                    NSNumber *lastFetchStatus = [[NSNumber alloc]
+                        initWithInt:[self mapLastFetchStatus:(FIRRemoteConfigFetchStatus)
+                                                                 [remoteConfig lastFetchStatus]]];
+                    NSMutableDictionary *resultDict = [[NSMutableDictionary alloc] init];
+                    resultDict[@"LAST_FETCH_TIME"] = lastFetchTime;
+                    resultDict[@"LAST_FETCH_STATUS"] = lastFetchStatus;
+
+                    if (status != FIRRemoteConfigFetchStatusSuccess) {
+                      FlutterError *flutterError;
+                      if (status == FIRRemoteConfigFetchStatusThrottled) {
+                        int mills =
+                            [[error.userInfo
+                                valueForKey:FIRRemoteConfigThrottledEndTimeInSecondsKey] intValue] *
+                            1000;
+                        resultDict[@"FETCH_THROTTLED_END"] = [[NSNumber alloc] initWithInt:mills];
+                        NSString *errorMessage =
+                            @"Fetch has been throttled. See the error's FETCH_THROTTLED_END "
+                             "field for throttle end time.";
+                        flutterError = [FlutterError errorWithCode:@"FETCH_FAILED_THROTTLED"
+                                                           message:errorMessage
+                                                           details:resultDict];
+                      } else {
+                        NSString *errorMessage =
+                            @"Unable to complete fetch. Reason is unknown "
+                             "but this could be due to lack of connectivity.";
+                        flutterError = [FlutterError errorWithCode:@"FETCH_FAILED"
+                                                           message:errorMessage
+                                                           details:resultDict];
+                      }
+                      result(flutterError);
+                    } else {
+                      result(resultDict);
+                    }
+                  }];
+  } else if ([@"RemoteConfig#activate" isEqualToString:call.method]) {
+    bool newConfig = [[FIRRemoteConfig remoteConfig] activateFetched];
+    NSDictionary *parameters = [self getConfigParameters];
+    result(@{@"newConfig" : [[NSNumber init] initWithBool:newConfig], @"parameters" : parameters});
+  } else if ([@"RemoteConfig#setDefaults" isEqualToString:call.method]) {
+    FIRRemoteConfig *remoteConfig = [FIRRemoteConfig remoteConfig];
+    NSDictionary *defaults = call.arguments[@"defaults"];
+    [remoteConfig setDefaults:defaults];
+    [[NSUserDefaults standardUserDefaults] setValue:[defaults allKeys] forKey:DEFAULT_KEYS];
```

Comment:
This can be removed using `allKeysFromSource`

Reply:
Done